server/common/deliverable/get.ts

```diff
@@ -0,0 +1,49 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { Sequelize } from "sequelize";
+
+import models from "db/models";
+
+import paginate from "helpers/paginate";
+import { resJsonMessage } from "helpers/res-json-message";
+
+interface propsWhere {
+  userId?: string | string[];
+  issueId?: string | number;
+}
+
+export default async function get(req: NextApiRequest, res: NextApiResponse) {
+  const { address, issueId } = req.query;
+  const where = {} as propsWhere;
+
+  if (address) {
+    const user = await models.user.findOne({
```

Comment:
this should be done via include on the actual deliverable call. Again, it should never return a 404 as the path exists, it's just empty